doc/us-en/setup/Install-OpenEdge-on-Linux.md

```diff
@@ -1,2 +1,161 @@
+```
+
+## Deploy OpenEdge
+
+### Preparation Before Deployment
+
+**Statement**:
+
++ The following is an example of the deployment and startup of OpenEdge on Ubuntu system. The deployment operations on other Linux distributions are basically the same as the following descriptions. It is assumed that the environment required for OpenEdge operation has been configured [before](#Environment-Configuration).
++ The Ubuntu system mentioned below is based on the Linux Ubuntu 4.15.0-29-generic version of the kernel with a CPU architecture of x86_64. Execute command `uname -ar` and get the result like this:
+
+![ubuntu kernel detail](../../images/setup/os-ubuntu.png)
+
+Starting OpenEdge containerization mode requires the running device to complete the installation and operation of Docker. You can install it by referring to [Steps above](#Install-Docker).
+
+### Deployment Process
+
+- **Step1**: [Download](../Resources-download.md) OpenEdge archive;
+- **Step2**: Open the terminal and enter the OpenEdge directory for decompression:
+	- execute command `tar -zxvf openedge-xxx.tar.gz`;
+- **Step3**: After the decompression operation is completed, enter the OpenEdge package directory in the terminal, open a new terminal at the same time, execute the command `docker stats`, display the running status of the container in the installed docker, and then execute the command `bin/openedge -w .`, respectively. Observe the contents displayed by the two terminals;
+- **Step4**: If the results are consistent, it means that OpenEdge has started normally.
+
+***Notice:*** The official download page only provides the docker mode executable file. If you want to run in process mode, please refer to [Build-OpenEdge-From-Source.md](./Build-OpenEdge-from-Source.md)
+
+### Start deployment
+
+As mentioned above, download OpenEdge archive from the [Download page](../Resources-download.md) first (also can compile from source, see [Build-OpenEdge-From-Source.md](./Build-OpenEdge-from-Source.md)), then open the terminal to enter OpenEdge directory for decompression. After successful decompression, you can find that the openedge directory mainly includes bin, etc, var, etc., as shown in the following picture:
+
+![OpenEdge directory](../../images/setup/openedge-dir.png)
+
+The bin directory stores the openedge executable binary file, the etc directory stores the configuration of OpenEdge, and the var directory stores the configuration and resources for the modules of OpenEdge.
+
+Then, open a new terminal and execute the command `docker stats` to view the running status of the container in the installed docker, as shown in the following picture:
+
+![view the docker containers status](../../images/setup/docker-stats-before.png)
+
+It can be found that the current system does not have a docker container running.
+
+Then, go to the decompressed OpenEdge folder, execute the command `bin/openedge -w .` in the another terminal, observe the log of OpenEdge startup, as shown below:
```

Comment:
Then, go to the decompressed folder of OpenEdge, execute the command `bin/openedge -w`. In the another terminal, observe the log of OpenEdge startup, as shown below: